src/Keboola/StorageApi/TableExporter.php

```diff
@@ -179,9 +168,53 @@ public function exportTable($tableId, $destination, $exportOptions)
             ));
             $fs->rename($tmpFilePath, $destination);
         }
-
         $fs->remove($tmpFilePath);
+    }
+
+    /**
+     *
+     * Process async export and prepare the file on disk
+     *
+     * @param $tableId
+     * @param $destination
+     * @param $exportOptions
+     * @throws Exception
+     */
+    public function exportTable($tableId, $destination, $exportOptions)
+    {
+        $this->exportTables([
+            [
+                'tableId' => $tableId,
+                'destination' => $destination,
+                'exportOptions' => $exportOptions
+            ]
+        ]);
+    }
 
-        return;
+    /**
+     * @param array $tables
+     * @throws Exception
+     */
+    public function exportTables($tables = array())
+    {
+        $exportJobs = [];
+        foreach ($tables as $table) {
+            if (!isset($table['tableId'])) {
```

Comment:
empty?

Reply:
to už bych nechal na storage, ať si s tím poradí?

Reply:
oboji (!isset/empty) je tu nevalidni a kdyz uz tu tu kontrolu mas, tak by mohla byt i uzitecna

Reply:
jak to myslíš, že to tu je nevalidní?

Reply:
je stejne spatne volat to jako `exportTables([[]])` nebo jako `exportTables([['tableId' => ]])`, jestli to teda ma validovat vstup, tak poradne. Jestli to ma jen validovat strukturu toho pole, tak bych to spis udelal pres array_keys (a povinny exportOptions), aby to odchytlo i `exportTables([['talbeId' => 'my-table' ...]])
jo a signatura by mela byt `public function exportTables(array $tables = array())`

Reply:
tak ale tohle všechno by mělo vyřešit `empty($table['tableId'])`, ne?